github/snapcraft.py

```diff
@@ -1,37 +1,126 @@
-#!/usr/bin/env python2
+#!/usr/bin/env/python2
 
-import urllib2
+import requests
 import urlparse
+import re
```

Comment:
It's common to sort the python imports alphabetically, grouping them by type: first python libs, then external libs, then libs local to the project. So, this should be:
```
import re
import urlparse

import requests
```